ci_framework/playbooks/ceph.yml

```diff
@@ -94,8 +94,8 @@
   gather_facts: true
   vars:
     ssh_network_range: 192.168.122.0/24
-    # storage_network_range: 172.18.0.0/24
-    # storage_mgmt_network_range: 172.20.0.0/24
+    storage_network_range: 172.18.0.0/24
+    storage_mgmt_network_range: 172.20.0.0/24
```

Comment:
Would you consider leaving these commented out so that it works by default without network isolation and passing these as overrides in the call to the playbook?